docs/tutorials/basic_ua_commands.md

```diff
@@ -0,0 +1,257 @@
+
+Let's enable `cis` on our LXD container by running:
+
+```console
+$ sudo ua enable cis
+```
+
+After running it. you should see an output similar to this one:
+
+```
+ne moment, checking your subscription first
+Updating package lists
+Installing CIS Audit packages
+CIS Audit enabled
+Visit https://security-certs.docs.ubuntu.com/en/cis to learn how to use CIS
+```
+
+We can confirm that `cis` is enabled now by running:
+
+```console
+$ ua status
+```
+
+And we should see:
+```
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cis           yes       enabled   Center for Internet Security Audit Tools
+esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
+fips          yes       n/a       NIST-certified core packages
+fips-updates  yes       n/a       NIST-certified core packages with priority security updates
+livepatch     yes       n/a       Canonical Livepatch service
+```
+
+We can see now that `cis` is marked as `enabled` on status.
+
+
+### Disable
+
+Let's suppose that you don't want a service anymore, we can also disable any service offering
+through UA. For example, let's disable the `cis` service we just enabled by running on the LXD
+container:
+
+```console
+$ sudo ua disable cis
+```
+
+After running that command, let's now run status to see what happened to `cis`:
+```
+SERVICE       ENTITLED  STATUS    DESCRIPTION
+cis           yes       disabled  Center for Internet Security Audit Tools
+esm-apps      yes       enabled   UA Apps: Extended Security Maintenance (ESM)
+esm-infra     yes       enabled   UA Infra: Extended Security Maintenance (ESM)
+fips          yes       n/a       NIST-certified core packages
+fips-updates  yes       n/a       NIST-certified core packages with priority security updates
+livepatch     yes       n/a       Canonical Livepatch service
+```
+
+We can see that `cis` status is back to disabled.
+
+```
+Note: the disable command doesn't uninstall any package that was installed by
+the service. The command only removes the access you have to the service, but it
+doesn't undo any configuration that was applied on the machine.
+```
+
+### Detach
+
+Finally, what if I don't want to be attached to an UA subscription any longer ?
+We can achieve that through the `detach` command:
+
+```console
+$ sudo ua detach
+```
+
+This command will disable all of the UA services on the machine for you and
+get rid of the subscription stored on your machine during attach.
+
+```
+Note: the detach command will also not uninstall any packages that were installed by
+any service enabled through UA.
+```
+
+### Final thoughts
+
+This tutorial has covered the 6 main commands of UA. If you need more advanced options to configure
+the tool, please take a look on [How to guides](./docs/howtoguides). If that still doesn't cover
+your needs, feel free to reach the UA team on `#ubuntu-server` on Libera IRC.
+
+Before we finish this tutorial, let's just delete our LXD container by running:
```

Comment:
Maybe it is useful to tell the user to run this on the machine, and not inside of the container where the previous commands were executed.